Hangman/src/Hangmanlogic.java

```diff
@@ -2,44 +2,62 @@
 
 public class Hangmanlogic {
 
-
-	private static String word = "Hyponeurikostiskadiafragmakontravibrationer";
-	private static String hideWord = "___________________________________________";
-	private static int count = 0;
+//	private static String word = "Hyponeurikostiskadiafragmakontravibrationer";
+//	private static String hideWord = "___________________________________________";
+// 	private static int count = 0;
 
 	public static void main(String[] args) {
 		Scanner reader = new Scanner(System.in);
 
-		while (hideWord.contains("_")&&count < 7  ) {
+		HangmanRound aRound = new HangmanRound();
+
+		runGame(aRound, reader);
+
+		HangmanRound aRoundtwo = new HangmanRound();
+
+		runGame(aRoundtwo, reader);
+
+		reader.close();
+
+	}
+
+	private static void runGame(HangmanRound aRound, Scanner reader) {
+
+		StaticWordselection.setWord(aRound);
+
+		while (aRound.getShowWord().contains("_") && aRound.getScoreCount() < StaticWordselection.getMaxScore()) {
```

Comment:
Yeah this works but its better to make a local var then using the static class in the statement
`
int maxScore=StaticWordselection.getMaxScore();

`
then use maxScore when needed